com.anydesk.Anydesk.json

```diff
@@ -47,11 +47,25 @@
         },
         {
             "name": "gtkglext",
+            "build-commands": [
+              "sed '/AC_PATH_XTRA/d' -i configure.in",
+             "./configure --prefix=/app --disable-static",
+              "make",
+              "make install"
+            ],
             "sources": [
                 {
-                    "type": "git",
-                    "url": "https://gitlab.gnome.org/Archive/gtkglext.git",
-                    "commit": "04131c5f69070b2f194f02edd45b0622ca03793c"
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/sourceforge/gtkglext/gtkglext-1.2.0.tar.bz2",
```

Comment:
We don't recommend sourcefourge hosting where possible due to unreliability. Could you try https://download.gnome.org/sources/gtkglext/1.2/gtkglext-1.2.0.tar.bz2 (I may fix this myself in manual testing)